documentation/book/proc-resizing-persistent-volumes.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='proc-resizing-persistent-volumes-{context}']
+= Resizing persistent volumes
+
+You can provision increased storage capacity by increasing the size of the persistent volumes used by an existing {ProductName} cluster.
+Resizing persistent volumes is supported in clusters that use either a single persistent volume or multiple persistent volumes in a JBOD storage configuration.
+
+NOTE: You can increase but not decrease the size of persistent volumes. Decreasing the size of persistent volumes is not currently supported in {ProductPlatformName}.
+Decreasing the size of persistent volumes is currently not possible.
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster with support for volume resizing
+* The Cluster Operator is running.
+* A Kafka cluster using persistent volumes created using a storage class
```

Comment:
Can we combine the bullets here?

* A Kafka cluster using persistent volumes created using a storage class that supports volume expansion.